docs/user-guide/src/capabilities.md

```diff
@@ -0,0 +1,6 @@
+# Prusti capabilities summary
+
+This chapter documents capabilities and limitations of Prusti:
+
+- [Limitations of Prusti](capabilities/limitations.md)
+- [Supported Rust and verification features of Prusti](capabilities/supported.md)
```

Comment:
These are not in the same order as in `SUMMARY`, swap; also make the titles match everywhere.